src/rules/all-yarn-modern-files-conform.test.ts

```diff
@@ -49,7 +49,7 @@ describe('Rule: all-yarn-modern-files-conform', () => {
       });
 
       expect(result).toStrictEqual({
-        passed: true,
+        status: 'passed',
```

Comment:
All rules return a result object, and that result object contains the status. So the test file for every rule needed to be updated to check for the correct property.